public/student.html

```diff
@@ -240,11 +244,11 @@ <h1 id="brian">Brian &#xC2B9;&#xD0DD; Kim <span style="font-size: 1rem;">(he/him
       </div>
       <div class="col-lg-6 text-left pl-lg-5 pl-1 pt-lg-0 pt-2 py-5 d-flex flex-column justify-content-lg-around">
          <div class>
-            <h1 id="sasha">Sasha Peterson</h1>
+            <h3 class="h1" id="sasha">Sasha Peterson</h3>
             <p class="lead border-dark border-bottom">The University of Texas at El Paso, Environmental Science &amp; Engineering</p>
             <p>At the nexus of land, sea, cryosphere, and atmosphere, Arctic coastal zones are likely the most critical in terms of rapid change as well as its vitality to local communities. Nearly half of the Beaufort coast is comprised of sheltered shorelines, and despite this protection erosion is rampant. My research aims to assess the drivers of erosion in Arctic lagoons and determine its associated impacts. The overarching goals of my research are: (1) determine spatiotemporal trends of erosion since the late 1940s in Beaufort coastal lagoons, (2) determine the transport and timing of thawing sediments from bluff face to lagoon, (3) improve spatiotemporal resolution of land-ocean transfer and fate of carbon via erosion, and (4) determine the role that the winter season plays on physiochemical properties and vulnerability of Arctic coasts. My research utilizes in situ soil collection, lab experiments, time-lapse imagery, and novel remote sensing using UAVs and satellite imagery.</p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:speterson2@miners.utep.edu"><i class="  far fa-paper-plane"></i>  speterson2@miners.utep.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://www.researchgate.net/profile/Sasha_Peterson4/research" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://www.researchgate.net/profile/Sasha_Peterson4/research" target="_blank" aria-label="open page in new tab">Publications</a></button>
```

Comment:
specify what you're opening in new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5